index.js

```diff
@@ -2,9 +2,36 @@
  * The function below has no parameters. Instead, access the arguments from the command line.
  * The first argument passed after the filename should be either "plus" or "minus", which represents the kind of calculation that will be done. Every argument afterwards should be a number.
  * Depending on the operation, either add up all of the numbers or subtract all of the numbers, from left to right.
- * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the arguments added to the command line.
+ * @returns {number} The result of either adding all numbers or subtracting all numbers, depending on the 
+ * arguments added to the command line.
  */
-function calculator() {}
+function calculator() {
+   console.log(process.argv)
+if(process.argv[2] === undefined){
+
+    return "No operation provided..."
+}else if (process.argv.length <= 3){
+
+    return "No numbers provided..."
+}else if(!(process.argv[2].includes("plus") || process.argv[2].includes("minus")))
+
+    return `Invalid operation: ${process.argv[2]}`
+
+let sum = Number(process.argv[3])
```

Comment:
Since you can add or subtract to the number a more accurate variable name might be something like `total` or `result`